.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @0x656b694d @tommartensen @vikin91
+* @stackrox/release-improvers
```

Comment:
Is this correct? I see it like this

![Screenshot 2023-08-31 at 10 53 08](https://github.com/stackrox/actions/assets/114479/342c70e5-4ba8-466b-9117-dcc575bc21a5)

Reply:
Ah, I see your TODO now in the PR description - all good then!

Reply:
see todo in PR description :)